src/main/java/de/fearnixx/jeak/teamspeak/voice/sound/TeamspeakFastMixerSink.java

```diff
@@ -0,0 +1,417 @@
+package de.fearnixx.jeak.teamspeak.voice.sound;
```

Comment:
Please revisit and replace quirky logging statements & stuff.  
If possible, please provide the URL to the original source.